### DISCUSSION TITLE
Help translate Gephi to your language

### DISCUSSION THREAD

**Discussion by mbastian** [gephi/gephi#2414] [Announcements]
Gephi is partially translated in some languages, for ex:
• Dutch (44%)
• Catalan (63%)
• Chinese Traditional (33%)
• Hebrew (30%)
• Italian (32%)
• Turkish (32%)

Since, we recently migrated the project to [Weblate](https://hosted.weblate.org/projects/gephi/#languages) it should be easy enough to get started.

Help us if you can! We include any language > 90% in the upcoming release.

**Comment by ludvig55** [comment]
I partially translated the Gephi-Lite language file into Hungarian. I would like to try what it looks like. There was something I couldn't (interpret-what is the correct term-). I would put it in the program to understand what it should be translated to. I would like to ask for guidance on this. (I'm not a programmer, just an "enthusiastic" beginner user.
How should the Hungarian language file (hu.json) be formatted and where can it be pasted?
I would download this from the translation page:
Route: Gephi-gephi-lite-Hungarian
Which download format and extension should be answered.
Or what is the process of inserting it into the program? I would like your help.
Windows 10 64 bit Gephi 0.10.1
Thanks.
keklala@gmail.com

**Reply by mbastian** [reply]
Hi @ludvig55, thanks for your contributions to the localization. We appreciate it!

The integration between Weblate and GitHub is automated, so that you don't have to do anything. Once we release a new snapshot version, you can then download a development version of Gephi for your system and try out with the localisation you provided. I'll let you know when a new dev version is released.

**Reply by mbastian** [reply]
A new version has been released, try it out here [gephi-0.10.2-SNAPSHOT-windows-x64.exe](https://oss.sonatype.org/service/local/artifact/maven/content?r=snapshots&g=org.gephi&a=gephi&v=0.10.2-SNAPSHOT&c=windows-x64&p=exe) (Windows)

**Comment by ludvig55** [comment]
Hi Mbastian!
You make my day. Thanks. I see that there is still something to translate and improve because it is still incomplete. Greetings @ludvig55

**Reply by mbastian** [reply]
Cool, then just keep doing what you're doing 👍  As it looks like the translation is almost complete we could add it to the list of officially supported languages!
